Library/Formula/nsd.rb

```diff
@@ -24,12 +22,16 @@ def install
       --with-libevent=#{Formula["libevent"].opt_prefix}
     ]
 
-    args << '--enable-root-server' if build.with? 'root-server'
-    args << '--enable-bind8-stats' if build.with? 'bind8-stats'
-    args << '--enable-ratelimit' if build.with? 'ratelimit'
-    args << '--enable-zone-stats' if build.with? 'zone-stats'
+    args << "--enable-root-server" if build.with? "root-server"
+    args << "--enable-bind8-stats" if build.with? "bind8-stats"
+    args << "--enable-ratelimit" if build.with? "ratelimit"
+    args << "--enable-zone-stats" if build.with? "zone-stats"
 
     system "./configure", *args
-    system "make install"
+    system "make", "install"
+  end
+
+  test do
+    system "#{sbin}/nsd", "-v"
```

Comment:
I suppose that is possible. How do I get the test block to kill the process after a successful test? Or better still, can I run the daemon in the foreground, and have brew send it a TERM signal?

Just a curiosity: in the past, you've happily just accepted a version update. Why are you insisting on so many more tests now?